helix-core/Cargo.toml

```
@@ -49,6 +49,7 @@ chrono = { version = "0.4", default-features = false, features = ["alloc", "std"
 
 etcetera = "0.8"
 textwrap = "0.16.0"
+globset = "0.4.13"
```

I believe this was bumped to `0.4.14` in `helix-lsp` by dependabot a while ago, might as well keep this and the helix-lsp dep in-sync

```suggestion
globset = "0.4.14"
```